### ISSUE TITLE
Sprint 0 for MDOTS

### ISSUE THREAD

**Issue by jperk51** [department-of-veterans-affairs/va-mobile-app#6922]
Write up a sprint 0 for creating an MDOTS feature in the app. 

API: Medical Devices Ordering Tool (MDOT)
Description: Allows veterans to reorder medical supplies like hearing aid batteries.

There is already a service for this in vets-api, but as part of the sprint 0 prep you should poke/ask around to make sure that service is still viable.

**Comment by kpethtel** [comment]
Per discussion here, this is not ready for discovery yet: https://dsva.slack.com/archives/C05DFSM57FW/p1697220744272559

Here are my notes from the discovery I did: 
- product is owned by va-cto-supply-reordering
- that team is not aware of any documentation of the upstream service
- the vets-api has two functions for interacting with the upstream: 
1) MDOT::Client#get_supplies: A show method that seems to return all medical devices available to the user
  - web front end call occurs here:  src/applications/disability-benefits/2346/actions/index.js
  - accessible through the /v0/in_progress_forms/mdot
  - does not seem to work on staging. I've tried it in multiple ways, and they all showed 503 from upstream.
2) MDOT::Client#submit_order: posts a request for medical device for user
  - web front end call occurs here: src/applications/health-care-supply-reordering/config/form.js
  - accessible through the vets-api endpoint: /v0/mdot/supplies
  - uses the following body:
```
use_veteran_address,
use_temporary_address,
additional_requests,
vet_email,
order: [:product_id],
permanent_address: address_params,
temporary_address: address_params
```

where address params look like:
```
street
street2
city
state
country
postal_code
```
In other words, the user specifies a list of requested device ids and whether to send to a permanent or temporary address.

**Comment by jperk51** [comment]
@kpethtel the purpose of this ticket was to write an actual sprint 0 document for this: https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/va-mobile-app/features/sprint-0/product-brief-template.md

you can see a lot of different examples in the ../features folder
